js/src/utils/urls.js

```diff
@@ -8,9 +8,15 @@ import { getNewPath } from '@woocommerce/navigation';
  */
 import { API_RESPONSE_CODES } from '.~/constants';
 
-const getStartedUrl = '/google/start';
-const dashboardPath = '/google/dashboard';
-const settingsPath = '/google/settings';
+export const pagePaths = {
+	getStarted: '/google/start',
+	setupMC: '/google/setup-mc',
+	setupAds: '/google/setup-ads',
+	dashboard: '/google/dashboard',
+	reports: '/google/reports',
+	productFeed: '/google/product-feed',
+	settings: '/google/settings',
+};
```

Comment:
❓ / 💅  

Maybe not for this PR (or also we want this to be explicit) but since all the paths has google as prefix and are just the `kebab-case` transformation of the key we could consider to have a getGLAPagePath function.

Reply:
Being explicit might be better for this since they have the same strings on the PHP side.